geo/src/algorithm/relate/mod.rs

```diff
@@ -81,11 +81,11 @@ macro_rules! relate_impl {
 ///
 /// # Examples
 ///
-/// ```no_run
```

Comment:
These were causing `cargo test` to fail as `no_run` still tries to compile the code.